apps/admin-gui/src/app/shared/components/dialogs/add-group-hierarchical-include-dialog/add-group-hierarchical-include-dialog.component.html

```diff
@@ -0,0 +1,41 @@
+<div class="{{theme}}">
+  <h1 mat-dialog-title>
+    {{'DIALOGS.ADD_GROUPS_HIERARCHICAL_INCLUSION.TITLE' | translate}}
+  </h1>
+  <div mat-dialog-content class="dialog-container">
+    <perun-web-apps-alert alert_type="info">
+      {{'DIALOGS.ADD_GROUPS_HIERARCHICAL_INCLUSION.INFO_START' | translate}}
+      {{parentVo.name}}
+      {{'DIALOGS.ADD_GROUPS_HIERARCHICAL_INCLUSION.INFO_END' | translate}}
+    </perun-web-apps-alert>
+    <perun-web-apps-immediate-filter
+      [placeholder]="'DIALOGS.ADD_GROUPS_HIERARCHICAL_INCLUSION.SEARCH'"
+      (filter)="applyFilter($event)">
+    </perun-web-apps-immediate-filter>
+    <perun-web-apps-groups-list
+      *ngIf="!loading"
+      [tableId]="tableId"
+      [groups]="groups"
+      [selection]="selected"
+      [filter]="filterValue"
+      [displayedColumns]="['select', 'id', 'name', 'description']"
+      [disableRouting]="true"
+      [noGroupsAlert]="'DIALOGS.ADD_GROUPS_HIERARCHICAL_INCLUSION.NO_GROUPS_FOUND_ALERT'"
+      theme="vo-theme">
+    </perun-web-apps-groups-list>
+    <mat-spinner *ngIf="loading" class="ml-auto mr-auto"></mat-spinner>
+  </div>
+  <div *ngIf="!loading" mat-dialog-actions>
+    <button (click)="close()" mat-flat-button class="ml-auto">
+      {{'DIALOGS.ADD_GROUPS_HIERARCHICAL_INCLUSION.CANCEL' | translate}}
+    </button>
+    <button
+      [disabled]="selected.selected.length === 0"
+      (click)="confirm()"
+      class="ml-2"
```

Comment:
This margin is not needed, instead some margin could be set on the cancel button.